server/docs/finalGrades.yaml

```diff
@@ -12,14 +12,14 @@ definitions:
     example: 1
   FinalGrade:
     type: number
-    description: The numerical grade of a student for an attainment.
+    description: The numerical final grade of a student.
     format: int32
     minimum: 0
     example: 5
   FinalGradeDate:
     type: string
     format: date
-    description: Date when attainment is completed (e.g., deadline or exam date)
+    description: Date when final grade was calculated
```

Comment:
From what I recall, this field exists because Sisu wants this information, and if so, then I don't think this is correct. It should be the date that the "course is completed," which isn't necessarily the same date as the date the grade was calculated.